hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -336,6 +352,16 @@ public long lastScannedEntity() {
 		return lastScannedEntity;
 	}
 
+	public void resetAutoRenewSummaryCounts() {
+		entitiesScannedThisSecond = 0L;
+		entitiesTouchedThisSecond = 0L;
+	}
+
+	public void updateAutoRenewSummaryCounts(int numScanned, int numTouched) {
+		entitiesScannedThisSecond += numScanned;
+		entitiesTouchedThisSecond += numTouched;
+	}
```

Comment:
This update doesn't make use of the time elapsed from the last consensus timestamp to the current consensus timestamp, as discussed.